CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,3 @@
-Code of conduct
-===============
+# Code of conduct
```

Comment:
I'd expect these files to be consistent across the repos...

Reply:
I'm not sure what you're asking here. Can you clarify please?

Reply:
Long term I'd like to have a bot that checks all the repos (because we'll end up with a lot of them) meet certain standards (have the correct licence file, code of conduct file etc). While it's semantically the same Markdown, the check would probably just compare the files.

Reply:
I see. You want me to change it back to using `=`. Not a problem. Given that there are more heading size options with `#` would it better use these instead?

Reply:
Could do, conversation for elsewhere. 😃